lib/typeahead.js

```diff
@@ -45,7 +45,7 @@ var classBase = React.createClass({
     });
   },
   onChange:function (ev) {
-    var val = ev.target.value;
+    var val = event.target.value;
```

Comment:
I'm sorry, this isn't an acceptable fix. Can you describe the problem in an issue and we can come up with a solution that is cross browser compatible.